packages/next-codemod/transforms/__testfixtures__/next-async-request-api-dynamic-apis/async-api-type-cast-02.output.js

```diff
@@ -15,13 +15,13 @@ draftMode().isEnabled) {
 }
 
 export function MyCookiesComponent() {
-  const c = /* Next.js Dynamic Async API Codemod: Manually await this call, if it's a Server Component */
+  const c = /* @next-codemod-error Manually await this call, if it's a Server Component */
```

Comment:
follow-up: `cookies()` are only available in RSC, right? Then we can drop the conditional and just tell people to `await`. Also need to check the other APIs.